src/domain/checkForAndPlaceOrder.ts

```diff
@@ -523,21 +528,15 @@ function _handleOrderBookError(
     // It's possible that an order has not enough allowance or balance.
     // Returning DONT_TRY_AGAIN would be to drastic, but we can give the WatchTower a break by scheduling next attempt in a few minutes
     // This why, we don't so it doesn't try in every block
-    if (
-      [
-        ApiErrors.INSUFFICIENT_ALLOWANCE,
-        ApiErrors.INSUFFICIENT_BALANCE,
-      ].includes(data?.errorType)
-    ) {
-      const nextPollTimestamp =
-        blockTimestamp + WAITING_TIME_SECONDS_FOR_NOT_BALANCE;
+    if (BACKPRESSURE_API_ERRORS.includes(data?.errorType)) {
+      const nextPollTimestamp = blockTimestamp + BACKPRESSURE_DELAY;
       return {
         result: PollResultCode.TRY_AT_EPOCH,
         epoch: nextPollTimestamp,
         reason: `Not enough allowance/balance (${
           data?.errorType
         }). Scheduling next polling in ${Math.floor(
-          WAITING_TIME_SECONDS_FOR_NOT_BALANCE / 60
+          BACKPRESSURE_DELAY / 60
```

Comment:
shoudn't be `BACK_REASSURE_DELAY`? 
Also, i think this name is too generic. Before it was informing what is this waiting for. this is just reducing the checks in the case the owner don't have balance

Now it looks like its a delay to adjust for a producer consumer mismatch. why this change?

Reply:
The idea here is that there may be other API errors of which we want to back-off hitting the API. Maybe `BACKOFF_DELAY` would be more appropriate?

Reply:
not too strong oppinion, but at least would be good to mention is for the API errors, I wouldn't know what `BACKOFF_DELAY` is if i see it. And this is only used also for errors related to balance and allowance. I checked the other errors, and for now they didn't make sense to me to do the same treatment. 

If there's other errors that need to add some delay, is not clear it will be the same time, or same treatment. So maybe we are just getting ahead of ourselves. 

anyways, fine with me either way

Reply:
Upon looking at this for a second time, I unified the configuration and typings for api errors that have specifiable / configurable delays, and api errors that should trigger try again in next block. I think it's in a bit of a more readable state now :smiley: